src/tools/FirmwareUpdater/FirmwareUpdater.pro

```diff
@@ -0,0 +1,114 @@
+#-------------------------------------------------
```

Comment:
Files .pro are NOT auto generated. Only .pro.user are auto generated by Qt Creator